hack/test-end-to-end-docker.sh

```diff
@@ -15,9 +15,8 @@ echo "[INFO] Starting containerized end-to-end test"
 
 unset KUBECONFIG
 
-TMPDIR="${TMPDIR:-"/tmp"}"
-BASETMPDIR="${BASETMPDIR:-${TMPDIR}/openshift-e2e-containerized}"
 setup_env_vars
+setup_temp_dir_vars "test/end-to-end/containerized/"
```

Comment:
Just sticking my paranoid brain in here.  If this is a privileged process writing to and not interaction with users, it is probably better to write the content under /run.  If this is for a user session that carry on.